AppSettingsManagerBff/AppSettingsManagerBff.Model/PermissionLevel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace AppSettingsManagerBff.Model;
+
+public enum PermissionLevel
```

Comment:
Not sure if we'll actually get to it before the end of the semester. But would be really cool to actually publish the models from the API and just install a nuget package here to get all of these models instead of having to duplicate them